network-api/networkapi/wagtailpages/pagemodels/buyersguide/campaign_page.py

```diff
@@ -0,0 +1,131 @@
+from networkapi.wagtailpages.utils import get_language_from_request
+
+
+class BuyersGuideCampaignPageDonationModalRelation(TranslatableMixin, Orderable):
+    page = ParentalKey(
+        'BuyersGuideCampaignPage',
+        related_name='donation_modal_relations',
+    )
+    donation_modal = models.ForeignKey(
+        'DonationModal',
+        null=False,
+        blank=False,
+        on_delete=models.CASCADE,
+        help_text='Choose existing or create new donation modal'
+    )
+
+    panels = [
+        SnippetChooserPanel('donation_modal'),
+    ]
+
+    class Meta(TranslatableMixin.Meta, Orderable.Meta):
+        verbose_name = 'Donation Modals'
+        verbose_name_plural = 'Donation Modals'
+
+
+class BuyersGuideCampaignPage(FoundationMetadataPageMixin, Page):
+    parent_page_types = ['BuyersGuideEditorialContentIndexPage']
+    subpage_types: list = []
+    template = 'pages/buyersguide/campaign_page.html'
+
+    cta = models.ForeignKey(
+        'CTA',
+        related_name='buyersguide_campaign_page_for_cta',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text='Choose one of our call-to-action snippets, or create a new one.'
+    )
+    header = models.CharField(
+        max_length=250,
+        blank=True
+    )
+    narrowed_page_content = models.BooleanField(
+        default=False,
+        help_text='For text-heavy pages, turn this on to reduce the overall width of the content on the page.'
+    )
+    body = StreamField(
+        block_types=(('accordion', customblocks.AccordionBlock()),
+                     ('paragraph', RichTextBlock(
+                         features=full_content_rich_text_options,
+                         template='wagtailpages/blocks/rich_text_block.html',
+                     )),
+                     ('card_grid', customblocks.CardGridBlock()),
+                     ('image_grid', customblocks.ImageGridBlock()),
+                     ('iframe', customblocks.iFrameBlock()),
+                     ('image', customblocks.AnnotatedImageBlock()),
+                     ('audio', customblocks.AudioBlock()),
+                     ('image_text', customblocks.ImageTextBlock()),
+                     ('image_text_mini', customblocks.ImageTextMini()),
+                     ('video', customblocks.VideoBlock()),
+                     ('linkbutton', customblocks.LinkButtonBlock()),
+                     ('looping_video', customblocks.LoopingVideoBlock()),
+                     ('pulse_listing', customblocks.PulseProjectList()),
+                     ('single_quote', customblocks.SingleQuoteBlock()),
+                     ('slider', customblocks.FoundationSliderBlock()),
+                     ('spacer', customblocks.BootstrapSpacerBlock()),
+                     ('airtable', customblocks.AirTableBlock()),
+                     ('datawrapper', customblocks.DatawrapperBlock()),
+                     ('typeform', customblocks.TypeformBlock()), )
```

Comment:
What do you think about unindenting these so they are only indented one more level than the wrapping parenthesis?

Reply:
Updated the formatting